swan-lake/learn/using-the-cli-tools.md

```diff
@@ -222,7 +221,7 @@ These powerful supporting tools extend Ballerina to various ecosystem technologi
 </tr>
 <tr>
 <td class="cCommand">help</td>
-<td class="cDescription">Prints the usage details of any Ballerina command (e.g., `ballerina help pull`).
+<td class="cDescription">Prints the usage details of any Ballerina command (e.g., `bal help pull`).
```

Comment:
@azinneera will it be okay to list the `help` command description under the **Other Commands** section?

Reply:
+1